apps/epic-react/README.md

```diff
@@ -122,7 +122,7 @@ To run MySQL (via Docker) execute the following command:
 
 ```shell
 pnpm db:start
-pnpm db:push
+DATABASE_URL="mysql://root@localhost:3309/kcd-products" pnpm db:push
```

Comment:
my database_url wasn't being loaded when trying to push so this explicitly passes the param

Reply:
do you define the db URL in `.env`?